docs/RUNNING-LOCALLY.md

```diff
@@ -37,3 +37,46 @@ cd ../
 > development environment testing. This file should be based on the
 > `appsettings.json` file on EdFi.DataManagementService.Api.Tests.Integration
 > folder.
+## Running Unit Tests and Generate Code Coverage Report
+To run the unit tests locally, execute the following command in a PowerShell
+terminal:
+
+```shell
+# From base directory
+./build.ps1 UnitTest -Configuration Debug
+```
+
+The previous command should generate two files in the base directory, which
+contain all the merged results from the unit tests execution.
+```
```

Comment:
Code blocks are like headers in that they too are better with space around them.

Reply:
Added